src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py

```diff
@@ -173,7 +173,7 @@ def _stream_logs(byte_size,
         # If no new data available but not complete, sleep before trying
         # to process additional data.
         if (_blob_is_not_complete(metadata) and start >= available):
-            time.sleep(5)
+            time.sleep(1)
```

Comment:
storage rps limit per account is 20k, it's probably ok considering the payload size. need to watch if impacting the requests on other artifacts on the same account. The worst case is when no update in 30 minutes if the build fails. We may consider adding a check on build status every minute or something like that.

Reply:
@northtyphoon I thought about this again and decided to introduce a linear backoff. I tried this several times and we just end up hitting storage too frequently which concerns me again after looking at metrics, I think something like this is a better intermediate. The curse of pull models. :( Please review again.

It'll do this:

- 3 tries @ 1 sec
- 3 tries @ 2 
- 3 tries @ 4
- 3 tries @ 8
- infinite tries @ 15 (MAX) until timeout

/cc @sajayantony